### PR TITLE
Add disclaimer to README for non aos-azure team members

### LINKED ISSUE #627: DNS wildcard not set up to point to a router instance.
I was handed off an azure OCP cluster to test the router performance installed by openshift/openshift-azure.

Routing was not set up after the install finished, and I had to manually add wildcard DNS entries to point to the first router instance (there were two routers installed, as we had two infra nodes).

Is there a way to point automatically to the first router?  Anything that we missed during the install?

/cc @akrzos

### PR DESCRIPTION
**Pull request by ehashman**
```release-note
NONE
```

Closes #627.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 ## Prerequisites
 
+1. **Subscription**. In order to use the OpenShift on Azure plugin, your Azure subscription must be whitelisted to use the service. If you do not use a valid subscription, you will not have access to the appropriate resource groups in order to successfully run code in this repository. Reach out to the #aos-azure team in [Slack][slack] with any questions about this.
```
Comment (thekad) [inline README.md]: nit: let's avoid overly large lines and split them to fit the rest of the document

Comment (ehashman) [reply README.md]: I would love to truncate lines to 80 characters, I kept this as one long line to be consistent with the rest of the document :smile:

Do you want me to go ahead and reformat all of it?

Comment (thekad) [reply README.md]: oh! huh the rest of the diff in here shows 80ish long lines... I guess whatever makes it more consistent with the rest of the doc :man_shrugging: if that's long lines, then disregard my comment.

Comment (jim-minter) [reply README.md]: @ehashman I confess I have looked for a lightweight, safe and painless auto-formatting tool that we could standardise all our docs files with; worse I have failed.  I strongly prefer 80 character lines too.

Comment (jim-minter) [reply README.md]: (not asking you to reformat this doc now though)

Comment (jim-minter) [inline README.md]: I think the point I was wanting to get at is: these instructions are written on the basis that you're an aos-azure contributor with access to the subscription.  If you're not, these instructions are not going to get you all the way right now.

Comment (ehashman) [reply README.md]: Added a note along those lines above.

File: README.md [hunk 2]
```diff
@@ -6,6 +6,16 @@
 
 ## Prerequisites
 
+Note that this README is targeted at AOS-Azure contributors. If you are not a
+member of this team, these instructions may not work as they will assume you
+have permissions that you may not have.
+
+1. **Subscription**. In order to use the OpenShift on Azure plugin, your Azure
```
Comment (jim-minter) [inline README.md]: The comments about whitelisting in this paragraph are not really true.  I'd remove the paragraph.

Comment (ehashman) [reply README.md]: Done.

**Pull request merged**